src/client/providers/jediHelpers.ts

```diff
@@ -49,8 +49,8 @@ export function extractSignatureAndDocumentation(definition: proxy.IAutoComplete
 }
 
 export function highlightCode(docstring: string): string {
-    // Add blank line
-    docstring = EOL + EOL + docstring + EOL + EOL;
```

Comment:
Oh, my wrong expression 😅. It's not `EOF`, just the end of the docstring.

To elaborate, taking this docstring as an example

```
>>> fun1()
a

>>> fun2()
b
```

There are two 'doctest' blocks above.
With the rule 'starting with `>>>` and ending with blank line', only `fun1` block will be matched (`fun2` will not because there is no blank line behind it).

So I add blank lines, then it becomes

```

>>> fun1()
a

>>> fun2()
b

```

Then I replace them using regexs (from reStructuredText syntax to Markdown syntax).

After the replacements (at the end of the function), those blank lines are not needed anymore.
So you don't need to add a `EOL` after you `trim()` it.